### PR TITLE
Title Updates

### LINKED ISSUE #94: Update document page title on route transition
- [x] Index => 'Aptible Dashboard'
- [x] Stack#show#apps => "${stack.handle} Apps - ${organization.name}"
- [x] Stack#show#apps#new => "Create an App - ${stack.handle}"
- [x] Stack#show#databases => "${stack.handle} Databases - ${organization.name}"
- [x] Stack#show#databases#new => "Create a Database - ${stack.handle}"
- [x] Stack#show#logging => "${stack.handle} Logging - ${organization.name}"
- [x] App#show => "${app.handle} - ${stack.handle}" (note: this route currently immediately transitions to App#show#services, so this title would never be seen)
- [x] App#show#services => "${app.handle} Services - ${stack.handle}"
- [x] App#show#domains => "${app.handle} Domains - ${stack.handle}"
- [x] App#show#domains#new => "Add a domain - ${app.handle} - ${stack.handle}"
- ~~[ ] App#show#domains#edit => "Edit ${domain.virtual_domain} - ${app.handle}"~~ (see #112)
- [x] App#show#activity => "${app.handle} Activity - ${stack.handle}"
- [x] App#show#deprovision => "Deprovision ${app.handle} - ${stack.handle}"
- [x] Database#show#activity => "${database.handle} Activity - ${stack.handle}"
- [x] Database#show#deprovision => "Deprovision ${database.handle} - ${stack.handle}"

### PR DESCRIPTION
**Pull request by bantic**
* Stack#show#logging title is "${stack.handle} Logging - ${organization.name}"
  * Title update for App#show#services
  * Title update for Activity for App and Database pages

Also:
  * stubApp stubs a stack for the app in addition to the app itself

fixes #94

@mixonic for you

### CODE REVIEW

File: app/app/route.js [hunk 1]
```diff
@@ -5,11 +5,16 @@ export default Ember.Route.extend({
     if (tokens.length === 0) {
       tokens.push(this.currentModel.get('handle'));
     }
-    var stack = this.currentModel.get('stack');
+
+    let stack = this.get('stack');
+    Ember.assert('App Route must have a stack', stack);
```
Comment (mixonic) [inline app/app/route.js]: `Ember.assert('App Route must have a stack', !!stack);` is preferred. It ensures only a boolean is passed, which lets the VM optimize.

File: app/app/route.js [hunk 2]
```diff
@@ -5,11 +5,16 @@ export default Ember.Route.extend({
...
     tokens.push(stack.get('handle'));
     return tokens.join(' - ');
   },
   afterModel: function(model){
-    return model.get('stack');
+    let route = this;
+    return model.get('stack').then(function(stack){
+      route.set('stack', stack);
```
Comment (mixonic) [inline app/app/route.js]: In `title:` you should be able to presume that `stack` has been loaded.

Where did `let stack = this.currentModel.get('stack')` fall down?

File: app/app/route.js (hunk 1 above)
Comment (bantic) [inline app/app/route.js]: Cool, I'll change this. This is a dev-only optimization because `Ember.assert` is stripped in production, right?

Comment (mixonic) [inline app/app/route.js]: From Ember production builds, not from application builds I don't think.

**Pull request merged**